yarn.lock

```diff
@@ -2,9 +2,9 @@
 # yarn lockfile v1
 
 
-"@procore/js-sdk-endpoints@^1.7.2":
-  version "1.7.2"
-  resolved "https://registry.yarnpkg.com/@procore/js-sdk-endpoints/-/js-sdk-endpoints-1.7.2.tgz#2e12ab60e9feaea2aa5192c1f0a7b87f320bb05a"
+"@procore/js-sdk-endpoints@^1.9.1":
+  version "1.9.0"
```

Comment:
I think we might want to `rm yarn.lock && yarn` since it still references itself in this file.